system/cpuid.c

```diff
@@ -1,12 +1,14 @@
 // SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2020-2022 Martin Whitaker.
+// Copyright (C) 2004-2022 Sam Demeulemeester.
 //
 // Derived from memtest86+ cpuid.h
-// (original contained no copyright statement)
+
 
 #include <stdint.h>
 
 #include "cpuid.h"
+#include "display.h"
```

Comment:
Why are you including display.h? The system library shouldn't have any dependencies on the application.

Reply:
Remnant of a debug function. Removed.